src/3d/qgs3dmapsettings.h

```diff
@@ -362,6 +362,21 @@ class _3D_EXPORT Qgs3DMapSettings : public QObject, public QgsTemporalRangeObjec
     //! Returns whether to display labels on terrain tiles
     bool showLabels() const { return mShowLabels; }
 
+    //! Sets whether eye dome lighting will be used
```

Comment:
```suggestion
    /**
     * Sets whether eye dome lighting will be used
     * \see eyeDomeLightingEnabled()
     * \since QGIS 3.18
     */
```
etc
(i.e. add the missing \since annotations and link setter to getter with a \see tag)